Tests/resources/directory/twig/subdirectory2/index.html.twig

```diff
@@ -0,0 +1,2 @@
+Some text
+{{ 'string to translate'|trans({}, 'Test.Translation.SecondSubDir') }}
```

Comment:
CamelCase within domain components is not allowed. It will work here but would set a bad example
```suggestion
{{ 'string to translate'|trans({}, 'Test.Translation.Secondsubdir') }}
```